From c518597dddb9b8a0b8d895b28ebb4792c8a32c6e Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Thu, 25 Aug 2022 14:52:30 +0200
Subject: [PATCH] test: bump the base VM memory to 768M

as with 512M some tests occasionally trip off OOM-killer (e.g.
TEST-64 + multipath).

(cherry picked from commit 6a9c4977683a30fcd36baf64e35255e9846028c6)

Related: #2155516
---
 test/TEST-36-NUMAPOLICY/test.sh | 2 +-
 test/test-functions             | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/TEST-36-NUMAPOLICY/test.sh b/test/TEST-36-NUMAPOLICY/test.sh
index 7909b1dce3..a0bfd4079a 100755
--- a/test/TEST-36-NUMAPOLICY/test.sh
+++ b/test/TEST-36-NUMAPOLICY/test.sh
@@ -9,7 +9,7 @@ TEST_NO_NSPAWN=1
 . "${TEST_BASE_DIR:?}/test-functions"
 
 if qemu_min_version "5.2.0"; then
-    QEMU_OPTIONS="-object memory-backend-ram,id=mem0,size=${QEMU_MEM:-512M} -numa node,memdev=mem0,nodeid=0"
+    QEMU_OPTIONS="-object memory-backend-ram,id=mem0,size=${QEMU_MEM:-768M} -numa node,memdev=mem0,nodeid=0"
 else
     QEMU_OPTIONS="-numa node,nodeid=0"
 fi
diff --git a/test/test-functions b/test/test-functions
index 8ea2f97b71..f9a80884e8 100644
--- a/test/test-functions
+++ b/test/test-functions
@@ -450,7 +450,7 @@ run_qemu() {
     qemu_options+=(
         -smp "$QEMU_SMP"
         -net none
-        -m "${QEMU_MEM:-512M}"
+        -m "${QEMU_MEM:-768M}"
         -nographic
         -kernel "$KERNEL_BIN"
         -drive "format=raw,cache=unsafe,file=$image"
